### ISSUE TITLE
Fails when using unity desktop launcher

### ISSUE THREAD

**Issue by mapkyca** [mailpile/Mailpile#1263]
On a vanilla Ubuntu 14.04 lts with unity desktop.

Launching from the console with ```mp``` script works fine. 

Created a .desktop launcher file for this same script. On execution it prompts twice for the super secret password (once to unlock the keychain for some reason, and then a second time in browser). On entering the password in the mailpile dialog in the browser, the submit gives you a no such server. 

It looks like somewhere between serving the auth page and the submit, the web server dies. But only when launched via the .desktop file.

This was my mums computer, so didn't have any time to debug or grab logs.

**Comment by BjarniRunar** [comment]
Closing this - this isn't an issue with Mailpile, it's an issue with an unknown .desktop script which we don't even have a copy of. :-)

**Comment by mapkyca** [comment]
I'll try and get a copy of the .desktop, but literally it was just calling the command line app. Seems to die when run this way, but is fine when run from the console.

I'm wondering whether unity is automatically closing the process after a timeout somewhere, possibly I need to explicitly background the task (which I think I did, but can't remember if I did).

I'll investigate further next time I go visit.
